server/main.go

```diff
@@ -55,7 +59,10 @@ func getWebHandler(activeFileManager *ActiveFileManager, fileStore FileStore) ht
 			} else if method == "PUT" {
 				// uploading a file
 				handlePutFile(res, req, path[0], activeFileManager)
+			} else if method == "OPTIONS" {
+				return
 			} else {
+				res.Header().Set("Allow", "GET, PUT, OPTIONS")
```

Comment:
Unless you've verified every browser today (and even then), it's better to follow the spec and serve some meaningful information when an OPTIONS request comes in. I haven't read into this too much, but it seems that the Allow header might be the response that browsers are looking for.